src/plugins/formulas/formulas.js

```diff
@@ -436,16 +436,22 @@ export class Formulas extends BasePlugin {
     const changedCellsSet = new Set(changedCells.map(change => stringifyAddress(change)));
 
     dependentCells.forEach((change) => {
+      const { row, col } = change.address;
```

Comment:
It seems that [address may be `undefined`](https://github.com/handsontable/handsontable/pull/8227/files#diff-75a3f65e47cad775c4d9cef1c98b6ef3ad0ad696c9aaa6b62eaf9fe45a09833cR448-R449). Will it work properly in that case? Shouldn't there be any extra conditional?

Reply:
⬆️  I agree with @wszymanski. Previously the coords destructing was secured by the `sheetId !== void 0`. In the case of the named expression, there can be unexpected behavior.

Reply:
Does this make sense? fb9730b